dali/operators/imgcodec/image_decoder.h

```diff
@@ -243,7 +255,9 @@ class ImageDecoder : public StatelessOperator<Backend> {
       pinned_alloc_.pinned_malloc = static_dali_pinned_malloc;
       pinned_alloc_.pinned_free = static_dali_pinned_free;
       pinned_alloc_.pinned_ctx = mm::GetDefaultResource<mm::memory_kind::pinned>();
-      pinned_alloc_.pinned_mem_padding = spec.GetArgument<int64_t>("host_memory_padding");
+      pinned_alloc_.pinned_mem_padding = decoder_params_.count("pinned_memory_padding") == 0 ?
```

Comment:
Is  the`pinned_memory_padding` different from `host_memory_padding`? I am asking because I don't see any other mentions of `pinned_memory_padding` apart from this place. Is there a way to specify it?

Reply:
I think it is the same, but named differently.

Reply:
Good catch

Reply:
it is hard to verify the effect of the parameter